saleor/payment/migrations/0036_auto_20220518_0732.py

```diff
@@ -0,0 +1,49 @@
+# Generated by Django 3.2.13 on 2022-05-18 07:32
+
+import django.contrib.postgres.fields
+from django.db import migrations, models
+
+
+def update_available_actions(apps, schema_editor):
+    TransactionItem = apps.get_model("payment", "TransactionItem")
+    transactions = TransactionItem.objects.filter(
+        available_actions__contains=["capture"]
+    )
+    transaction_to_update = []
```

Comment:
It is simple `for` as the API for creating transactions hasn't been released yet, but everyone who would use the `main` branch could have an issue without this.